EGI-CLI-Swift-S3.md

```diff
@@ -27,7 +27,18 @@ fedcloud token check
 
 Last command should return a valid token.
 
-## Use Openstack
+## Use Openstack without fedcloudclient to access a specific project storage
+
+It is possible to use `openstack` command through `fedcloudclient` to interact with
+Openstack object storage, as [documented here](https://docs.egi.eu/users/data/storage/object-storage/#access-via-rclone).
```

Comment:
Replace with
https://docs.egi.eu/users/data/storage/object-storage/#access-with-the-fedcloud-cli